src/chrome/DatasetCommitInfo.tsx

```diff
@@ -17,22 +17,25 @@ const DatasetCommitInfo: React.FC<DatasetCommitInfoProps> = ({
   dataset,
   small=false
 }) => {
+  console.log('dataset', dataset)
   return (
-    <div className={classNames({
+    <div className={classNames('truncate', {
       'text-sm': !small,
       'text-xs': small
     })}>
       <div className={classNames('text-black font-semibold flex justify-between items-center mb-2')}>
-        <div className='dataset_commit_info_text'>{dataset.commit?.title}</div>
-        <div className='flex-grow-0 text-qrigreen'>
-          <Icon icon='automationFilled' size={small ? 'xs' : 'sm'}/>
-        </div>
+        <div className='dataset_commit_info_text truncate flex-grow' title={dataset.commit?.title}>{dataset.commit?.title}</div>
```

Comment:
I find the text of this to be too bold. It competes with the title on the left. Can we drop the weight of the type?